### ISSUE TITLE
Some thumbnail packs are truncated (again)

### ISSUE THREAD

**Issue by tgm0** [libretro-thumbnails/libretro-thumbnails#53]
Hi.

Concerning:
http://thumbnailpacks.libretro.com/MAME.zip
http://thumbnailpacks.libretro.com/Nintendo%20-%20Super%20Nintendo%20Entertainment%20System.zip

The downloaded file can't be unzipped as the archive is damaged.
The size reported when the download starts matches the downloaded size, but it is smaller than the size displayed on the page.
It's not a network problem.

Theere may be more damages files, I just found the above on the few downloads I made.
Same issue was reported a few months back here [https://github.com/libretro-thumbnails/libretro-thumbnails/issues/47](url)

**Comment by RobLoach** [comment]
I assume you're on Windows. I'll do another search for Windows-breaking filenames.

It opens and extracts file for me (760.7MB). I'm on Linux.

**Comment by tgm0** [comment]
I am on windows indeed.
Tried again today and both sets downloaded fine; the download for the snes set today showed a total size of 726MB and MAME a full 3GB.
The other day snes showed 256MB when downloading and I did try with Firefox, Chrome and even Edge just to be sure it wasn't a browser issue of some sort.
If you guys have srcipts in place to upload this packs then maybe soimething got stuck and i was just unlucky to try and download at the wrong time.
Thanks for all you are doing and bye bye...
